src/stockwatch/use_cases/configuring.py

```diff
@@ -17,12 +18,14 @@ class DeGiroServerConfig(ConfigSectionBase):
     login_url: str = "https://trader.degiro.nl/login/secure/login"
     client_url: str = "https://trader.degiro.nl/pa/secure/client"
     portfolio_url: str = (
-        "https://trader.degiro.nl/reporting/secure/v3/positionReport/csv"
+        "https://trader.degiro.nl/portfolio-reports/secure/v3/positionReport/csv"
     )
     account_url: str = (
-        "https://trader.degiro.nl/reporting/secure/v3/cashAccountReport/csv"
+        "https://trader.degiro.nl/portfolio-reports/secure/v3/cashAccountReport/csv"
```

Comment:
These fixes also have to be done in `src/stockwatch/configuration/stockwatch.toml`